### PR TITLE
Add test for celery integration.

### PR DESCRIPTION
**Pull request by anmic**

### CODE REVIEW

**Comment by anmic** [comment]
@vmihailenco PTAL

File: pybrake/test_celery_integration.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+def test_celery_integration():
+    server_address = ('', 8080)
+    server = HTTPServer(server_address, Handler)
+
+    httpd_thread = threading.Thread(target=server.serve_forever)
+    httpd_thread.setDaemon(True)
+    httpd_thread.start()
+
+    try:
+        result = raiseException.delay()
```
Comment (vmihailenco) [inline pybrake/test_celery_integration.py]: use raiseException.apply to execute task locally - http://docs.celeryproject.org/en/latest/reference/celery.app.task.html

**Comment by anmic** [comment]
I use task.apply() method to execute task locally.

File: pybrake/test_celery.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from pybrake.celery import patch_celery
+from .notifier import Notifier
+
+app = Celery("test_celery", broker="redis://localhost", backend='redis://localhost')
+
+notifier = Notifier(host="http://localhost:8080", environment="celery")
+patch_celery(notifier)
+
+@app.task
+def raiseException():
```
Comment (vmihailenco) [inline pybrake/test_celery.py]: fix name

File: pybrake/test_celery_integration.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import json
+import threading
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from .test_celery import raiseException
```
Comment (vmihailenco) [inline pybrake/test_celery_integration.py]: group imports

**Comment by anmic** [comment]
Fixed imports.

**Pull request merged**